### DISCUSSION TITLE
How do i close all issues set as done in my projects (v2)

### DISCUSSION THREAD

**Discussion by yassinm** [community/community#35426] [Projects and Issues]
Hi All.

I have been using github issues with the new projects and its been amazing. Except one smal thing.  I am surprised there is no easy way to do this.

When i am working on an issue i usually juts drag and drop it as `Done` however i end up have 20 or so sitting in my Done column. I tried hard to find a way to set them all to closed at the end of the day. Right now i have to click close on each issue and it takes few seconds on each issue and its really starting to be annoying.

I could close the issue as soon as i am done but i would rather have it sitting there in the done section on the board and close them all in one shot. Is there an easy way i can close all of the items set as done to closed in one shot please ?

**Comment by yassinm** [comment]
Also i could not figure a way to search for all issues that are set as Done in a project from the issues search syntax. If that is at all possible i could then set all of them as closed by first searching all of them and them marking them as closed. 

tl;dr whats the search syntax to search for all issues that have the Done state in a projects board ?

**Reply by mnbf9rca** [reply]
`status:Done` - but i can't then see how to automate closing them...
